src/modules/tools/zprobe/CartGridStrategy.cpp

```diff
@@ -15,7 +15,8 @@
 
     The size of the grid can be set with...
 
-      leveling-strategy.rectangular-grid.size        7
+      leveling-strategy.rectangular-grid.grid_x_size        7
+      leveling-strategy.rectangular-grid.grid_y_size        7
```

Comment:
AFAIK this is backward compatible so the old config should continue to work. That was the intention to have it be totally backward compatible with no config changes needed.